### PR TITLE
Add new parameter 'byRowIndexes' in FILTERING transformer

### PR DESCRIPTION
**Pull request by AlenaBudevich**

### CODE REVIEW

File: vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java [hunk 1]
```diff
@@ -44,8 +44,11 @@
     private static final String BY_MAX_COLUMNS_PROPERTY = "byMaxColumns";
     private static final String BY_MAX_ROWS_PROPERTY = "byMaxRows";
     private static final String BY_COLUMNS_NAMES_PROPERTY = "byColumnNames";
+    private static final String BY_ROW_NUMBERS_PROPERTY = "byRowNumbers";
```
Comment (valfirst) [inline vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java:47]: `byRowIndexes` ?

File: vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java [hunk 2]
```diff
@@ -90,13 +98,22 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
...
+            return Optional.ofNullable(byMaxRows)
+                    .map(Integer::parseInt)
+                    .filter(m -> m < list.size())
+                    .map(m -> list.subList(0, m))
+                    .orElse(list);
+        }
+        else
+        {
+            return Stream.of(StringUtils.split(byRowNumbers, ';'))
+                    .map(Integer::valueOf).map(list::get).collect(Collectors.toList());
```
Comment (valfirst) [inline vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java:115]: `.mapToInt(Integer::parseInt).`

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — CHANGES_REQUESTED** [review]
All the docs data should go as a separate commit except the part explaining byRowIndexes

File: vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java [hunk 3]
```diff
@@ -90,13 +98,22 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
...
+            return Optional.ofNullable(byMaxRows)
+                    .map(Integer::parseInt)
+                    .filter(m -> m < list.size())
+                    .map(m -> list.subList(0, m))
+                    .orElse(list);
+        }
+        else
+        {
+            return Stream.of(StringUtils.split(byRowIndexes, ';')).mapToInt(Integer::parseInt)
+                    .mapToObj(list::get).collect(Collectors.toList());
```
Comment (uarlouski) [inline vividus-bdd-engine/src/main/java/org/vividus/bdd/transformer/FilteringTableTransformer.java:115]: please align in tree-like view

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**